odoo/tools/template_inheritance.py

```diff
@@ -104,7 +106,7 @@ def locate_node(arch, spec):
     return None
 
 
-def apply_inheritance_specs(source, specs_tree, inherit_branding=False, pre_locate=lambda s: True):
+def apply_inheritance_specs(source, specs_tree, inherit_branding=False, pre_locate=lambda s: True, invalid_xpath_list=None):
```

Comment:
This is a bit too specific and error prone.
I'd prefer passing a callback named "on_locate_failed" (or some other meaningful name).
What does @xmo-odoo think ?